examples/cloud-secrets/Earthfile

```diff
@@ -0,0 +1,30 @@
+VERSION 0.7
+PROJECT earthly-technologies/example
+FROM alpine:latest
+
+cloud-secrets:
+  # Note that earthly has changed it's implementation of secrets between v0.6 and v0.7
+  # This example is for the current 0.7 version
+  #
```

Comment:
Maybe I wouldn't include this little introduction, as 0.6 secrets were only experimental anyway.